ui/app/mainui/activitycenter/views/ActivityNotificationNewsMessage.qml

```diff
@@ -29,16 +29,28 @@ ActivityNotificationBase {
             Layout.alignment: Qt.AlignTop
             Layout.fillWidth: true
 
-            StatusMessageHeader {
-                Layout.fillWidth: true
-                displayNameLabel.text: root.notification ? root.notification.newsTitle : ""
-                timestamp: root.notification ? root.notification.timestamp : 0
-            }
+            RowLayout {
+                Layout.maximumWidth: parent.width
+                Layout.maximumHeight: 22
```

Comment:
Is this `maximumHeight` height necessary here? It's kind of magic number which is not good for layouts flexibility / maintainability. If I understand correctly it's not needed bc both text and timestamp are single-line and the height should be based on their implicit height.

Reply:
You're right. I thought it was necessary to put a height for it to elide and not wrap, but I not remember that you need to specify the `wordWrap` property for it to do that.
Removed.